climada/util/api_client.py

```diff
@@ -302,8 +312,8 @@ def get_dataset(self, data_type=None, name=None, version=None, properties=None,
             raise Client.NoResult("there is no dataset meeting the requirements")
         return jarr[0]
 
-    def get_dataset_by_uuid(self, uuid):
-        """Returns the data from 'https://climada/rest/dataset/{uuid}' as DatasetInfo object.
+    def get_dataset_info_by_uuid(self, uuid):
+        """Returns the data from 'https://climada.ethz.ch/data-api/v1/dataset/{uuid}' as DatasetInfo object.
```

Comment:
Will this have to be updated with each release of the api? That looks like a road to have inconsistent docstrings...

Reply:
True, it may lead to inconsistencies but it's still useful to see a valid address in the docs, rather than a reference to the config file or pattern like '...api/vXY/dataset...', up for guessing.

In any case I don't expect many updates in the api url, and only if the client itself has major updates in the DatasetInfo class a change of that url is going to be required. (note that the version v1 is there to stay, new versions will not replace just amend)